app/src/test/java/gov/va/vro/service/BipClaimServiceTest.java

```diff
@@ -80,7 +79,7 @@ void hasAnchors() throws BipException {
                 createContention(List.of(claimProps.getSpecialIssue1(), "OTHER"))));
 
     BipClaimService claimService = new BipClaimService(claimProps, bipApiService, null, null);
-    assertTrue(claimService.hasAnchors(bipClaimId));
+    assertFalse(claimService.hasAnchors(bipClaimId));
```

Comment:
Did you mean to leave this in "hasAnchors"? or should there be two methods one for false and one for true? I am mildly confused. But it may just need a name change.....

Reply:
I need to add another test for that. I'll keep track of it and add it to a different PR where I will also add a new E2E test